web/cobrands/sass/_report_list.scss

```diff
@@ -152,32 +152,16 @@ input.bulk-assign {
   padding: 1rem;
 
   .filter-group {
-    display: flex;
-    flex-direction: row;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(10rem, 1fr));
```

Comment:
This looks a bit funny here (right margin of top row compared to underneath), that doesn't happen on live:
<img width="478" alt="Screenshot 2024-08-29 at 17 57 15" src="https://github.com/user-attachments/assets/b2f7b59c-0abc-4cc4-9566-00e4aa65923d">
Perhaps the `multi-select-button, .govuk-select` needs a width of 100%? Not sure I've understood it fully.